<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>actionSettingsDialog</class>
 <widget class="QDialog" name="actionSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Action settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mainwin.qrc">
    <normaloff>:/mainWin/logo.png</normaloff>:/mainWin/logo.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>78</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Every X days</string>
   </property>
   <widget class="QCheckBox" name="runEveryXDaysCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>22</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Run script every</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>25</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>days.</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="daysSpin">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>20</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>138</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Commands time interval</string>
   </property>
   <widget class="QSpinBox" name="intervalSpin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>22</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999999999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>24</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ms</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="timeGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>18</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Time settings</string>
   </property>
   <widget class="QSpinBox" name="hourSpin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>23</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="minSpin">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>59</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="secSpin">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>59</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>22</y>
      <width>3</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>22</y>
      <width>2</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>101</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Run on</string>
   </property>
   <widget class="QCheckBox" name="mondayCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Monday</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="tuesdayCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tuesday</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="wednesdayCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Wednesday</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="thursdayCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Thursday</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fridayCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Friday</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="saturdayCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Saturday</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="sundayCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sunday</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="everydayCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Every day</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="mainwin.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>applyButton</sender>
   <signal>clicked()</signal>
   <receiver>actionSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>actionSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
